recipes/capnproto/all/conanfile.py

```diff
@@ -100,7 +102,7 @@ def validate(self):
 
     def build_requirements(self):
         if self.settings.os != "Windows":
-            self.tool_requires("libtool/2.4.7")
+            self.tool_requires("libtool/[>=2 <3]")
```

Comment:
```suggestion
            self.tool_requires("libtool/2.4.7")
```
It is better that this keeps aligned with the rest of the recipes in ConanCenter that uses 2.4.7.
PRs should be focused on doing 1 thing, in this case, adding the ``lite_mode`` option, and this seems unrelated.

Reply:
Done by dropping the commit that made this change, thanks!